app/scripts/lib/url.js

```diff
@@ -89,6 +89,16 @@ function (_) {
       return origin;
     },
 
+    /**
+     * Returns true if given "uri" has HTTP or HTTPS scheme
+     *
+     * @param {String} uri
+     * @returns {boolean}
+     */
+    isHTTP: function (uri) {
```

Comment:
`isUri()`? I'm guessing a `uri` couldn't be protocol-relative URL, like "//mozilla.com/" (http://www.paulirish.com/2010/the-protocol-relative-url/)